main/src/cgeo/geocaching/export/PersonalnoteExport.java

```diff
@@ -0,0 +1,84 @@
+package cgeo.geocaching.export;
+
+import cgeo.geocaching.CgeoApplication;
+import cgeo.geocaching.R;
+import cgeo.geocaching.activity.ActivityMixin;
+import cgeo.geocaching.connector.ConnectorFactory;
+import cgeo.geocaching.connector.IConnector;
+import cgeo.geocaching.connector.capability.PersonalNoteCapability;
+import cgeo.geocaching.models.Geocache;
+import cgeo.geocaching.utils.AsyncTaskWithProgress;
+import cgeo.geocaching.utils.Log;
+
+import android.app.Activity;
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Exports personal notes.
+ */
+public class PersonalnoteExport extends AbstractExport {
+
+    private int persNotesCount = 0;
+
+    public PersonalnoteExport() {
+        super(R.string.export_persnotes);
+    }
+
+    @Override
+    public void export(@NonNull final List<Geocache> cachesList, @Nullable final Activity activity) {
+        final Geocache[] caches = cachesList.toArray(new Geocache[cachesList.size()]);
+        new ExportTask(activity).execute(caches);
+    }
+
+    private class ExportTask extends AsyncTaskWithProgress<Geocache, Boolean> {
+        /**
+         * Instantiates and configures the task for exporting personal notes.
+         */
+        ExportTask(@Nullable final Activity activity) {
+            super(activity, getProgressTitle(), CgeoApplication.getInstance().getString(R.string.export_persnotes), true);
+        }
+
+        @Override
+        protected Boolean doInBackgroundInternal(final Geocache[] caches) {
+            try {
+                for (final Geocache cache : caches) {
+                    final IConnector connector = ConnectorFactory.getConnector(cache);
+                    if (connector instanceof PersonalNoteCapability && StringUtils.isNotBlank(cache.getPersonalNote())) {
+                        publishProgress(++persNotesCount);
```

Comment:
@Bananeweizen at first I haven't seen no progress bar. But after your comment I looked closer. I've copied the `indeterminate=true` from the `FieldnoteExport.ExportTask`. After changing this to false (or using the default false) I now have a progress bar. Now publishing the progress on every Geocache makes sense. It's not very smooth on my Device, but much more informative to see the overall progress.